<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>861</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>350</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="upload_folder_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>160</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload Converted </string>
       </property>
      </widget>
      <widget class="QPushButton" name="upload_everything_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>160</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload Everything</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string> Check to enable upload all button</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QListView" name="list_view">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>501</width>
         <height>571</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="uploaded_files_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>640</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of files: </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>36</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Available Converted files</string>
       </property>
      </widget>
      <widget class="QPushButton" name="previous_btn">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>670</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="shortcut">
        <string>A</string>
       </property>
      </widget>
      <widget class="QPushButton" name="next_btn">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>670</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="shortcut">
        <string>D</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
